### PR TITLE
[mercedesme] Additional mappings and channels

### LINKED ISSUE #17266: [mercedesme] Duplicate channels mercedesme:bev:mercedesme_account:...hvac#aux-status
Moved from https://github.com/weymann/MercedesMe-OH-Drops/issues/3 submitted by @lucbaert

My vehicle has been "discovered", I can see it inthings, but when I create an item for e.g. Electric Range, I get an error. In openhab.log I see:
```
2024-08-06 16:45:23.771 [ERROR] [ore.thing.internal.ThingRegistryImpl] - Could not inform the ThingTracker 'org.openhab.core.thing.internal.ThingManagerImpl@3b801bdd' about the 'THING_UPDATED' event!
java.lang.IllegalArgumentException: Duplicate channels mercedesme:bev:mercedesme_account:XYZ:hvac#aux-status
```
I am using openhab 4.2.0 and installed

### PR DESCRIPTION
**Pull request by weymann**
Fixes #17266 

- **Mappings added**
   - service#starterBattery
   - tires#sensor-available
   - tires#marker*
  
- **Channels added** 
  see [readme](https://github.com/weymann/MercedesMe-OH-Drops/blob/main/README.md)
  - `eco` group *(new)*
    -  `accel` - Acceleration Score
    - `coasting`- Coasting Score
    - `constant` - Constant Driving Score
    - `bonus` - Bonus range towards *very sportive driver*

  - `hvac` group
    - `ac-status` - Status of preconditioning command
    - `aux-status` - Status of auxiliary heating command

  - `charge` group
    - `status` - Status of charging
    - `error` - In case of problem description of error 

  - `position` group
    - `status` - Positioning status

### CODE REVIEW

File: bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/eco-channel-types.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	</channel-type>
+	<channel-type id="constant">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Constant Score</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="bonus">
+		<item-type>Number:Length</item-type>
+		<label>Bonus Range</label>
+		<state pattern="%d km" readOnly="true"/>
```
Comment (lsiepel) [inline bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/eco-channel-types.xml:24]:
```suggestion
		<state pattern="%d %unit%" readOnly="true"/>
```

Doublecheck: when updating the channel, the default is m, make sure it is converted properly.

File: bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/hvac-channel-types.xml [hunk 2]
```diff
@@ -10,14 +10,28 @@
 	</channel-type>
 	<channel-type id="temperature-hvac">
 		<item-type>Number:Temperature</item-type>
-		<label>AC Temperature</label>
+		<label>AC Desired Temperature</label>
```
Comment (lsiepel) [inline bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/hvac-channel-types.xml:13]:
```suggestion
		<label>AC Setpoint</label>
```
(for your consideration)

**Review by lsiepel — CHANGES_REQUESTED** [review]
Overall LGTM. Very minor comments.
The thing upgrade instructions are missing for the eco group. the other new and changed channels are added dynamically so they don;t need instructions.

File: bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/eco-channel-types.xml (hunk 1 above)
Comment (weymann) [reply bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/eco-channel-types.xml:24]: Very good hint. Forgot to add channel to `MetaDataAdjuster`.

File: bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/hvac-channel-types.xml (hunk 2 above)
Comment (weymann) [reply bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/hvac-channel-types.xml:13]: True - changed to `AC Setpoint Temperature`

**Comment by weymann** [comment]
> The thing upgrade instructions are missing for the eco group. the other new and changed channels are added dynamically so they don;t need instructions.

Added `instructions.xml` in `OH-INF/update` for added and updated channels: Please double check because till now I wasn't familiar with that.

**Comment by lsiepel** [comment]
> > The thing upgrade instructions are missing for the eco group. the other new and changed channels are added dynamically so they don;t need instructions.
> 
> Added `instructions.xml` in `OH-INF/update` for added and updated channels: Please double check because till now I wasn't familiar with that.

They are fine. Only thing to add is that you also need to set the proiperty `thingTypeVersion` for each thing in their xml structure.
```
		<properties>
			<property name="thingTypeVersion">1</property>
		</properties>
````

**Comment by weymann** [comment]
> > > The thing upgrade instructions are missing for the eco group. the other new and changed channels are added dynamically so they don;t need instructions.
> > 
> > 
> > Added `instructions.xml` in `OH-INF/update` for added and updated channels: Please double check because till now I wasn't familiar with that.
> 
> They are fine. Only thing to add is that you also need to set the proiperty `thingTypeVersion` for each thing in their xml structure.
> 
> ```
> 		<properties>
> 			<property name="thingTypeVersion">1</property>
> 		</properties>
> 
I really need some help here. [Documentation available](https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types) is not really covering all possible problems so I am  guessing around what's the right instructions.
I've a similar problem with https://github.com/openhab/openhab-addons/issues/17265 where I try out different solutions but the tutorial above doesn't provide any hint to solve it.

**Comment by lsiepel** [comment]
As you managed to fix the other PR, can you fix the conflict and proceed or do you need some specific information to continue. Let me now an i'll do the best i can.

**Comment by weymann** [comment]
> As you managed to fix the other PR, can you fix the conflict and proceed or do you need some specific information to continue. Let me now an i'll do the best i can.

Yes, looks fine now. Increased `thingTypeVersion` for due to new instructions, testing successful.
I'm good to go!

**Review by lsiepel — APPROVED** [review]
LGTM

**Pull request merged**